examples/header-group.html

```diff
@@ -0,0 +1,80 @@
+<template id=header-group>
+
+<style>
+
+  header-group
+    { display: block }
+
+  header-group
+    hgroup
+      { background: #bada55; color: var(--darkest-color) }
+
+</style>
+
+<hgroup>
+  <h1><a href=/examples/index.html>snuggsi ツ</a></h1>
+  <slot name=header><h2>try an example</h2></slot>
+</hgroup>
+
+</template>
+
+<script>
+const tag = `header-group`
+
+Element `header-group`
+
+(class extends HTMLElement {
+
+  static onimportload (event) {
```

Comment:
@brandondees After checking seems there is an `onload` reserved for `window` and `img` Could be a situation of specs out of date as there is an `onload` for `<link>` now as well. Perhaps we can have the default `onload` be for respective `link rel=import` however we may want to reserve this event for when the document is done loading. Perhaps that's not a concern of ours.

Please advise.

GlobalEventHandler.onload event - https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onload

GlobalEventHandler.on* events - https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers